imports/ui/ledger/LedgerActions.jsx

```diff
@@ -924,7 +924,7 @@ class WithdrawButton extends LedgerButton {
             if (res) {
                 if (this.props.address) {
                     res.value.msg.push({
-                        type: 'cosmos-sdk/MsgWithdrawValidatorCommission',
+                        type: 'cosmos-sdk/MsgWithdrawValidatorRewardsAll',
```

Comment:
The message is never changed. https://github.com/cosmos/cosmos-sdk/blob/v0.38.5/x/distribution/types/msg.go#L90